lib/src/main/java/uk/gov/di/ipv/cri/passport/library/domain/verifiablecredential/Evidence.java

```diff
@@ -1,19 +1,47 @@
 package uk.gov.di.ipv.cri.passport.library.domain.verifiablecredential;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
 import uk.gov.di.ipv.cri.passport.library.annotations.ExcludeFromGeneratedCoverageReport;
 
+import java.util.List;
+
+import static uk.gov.di.ipv.cri.passport.library.domain.verifiablecredential.VerifiableCredentialConstants.EVIDENCE_TYPE_IDENTITY_CHECK;
+
 @DynamoDbBean
 @ExcludeFromGeneratedCoverageReport
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class Evidence {
+
+    private String type = EVIDENCE_TYPE_IDENTITY_CHECK;
+    private String txn;
     private int strength;
     private int validity;
```

Comment:
The output JSON needs to have these values be named `strengthScore` and `validityScore`. We will need to update to match. Probably easiest to change the var names and getter and setter names.

Reply:
Yep - that's a good spot. I've made a sneaky ninja edit so we should be good now.